.pipelines/ci.yml

```diff
@@ -851,10 +851,9 @@ stages:
         architecture: 'x64'
       displayName: "Use Python 3.12"
 
-    # iOS xcframework build doesn't work with CMake 3.25.1, pin to 3.25.0
     - script: |
-        python -m pip install cmake==3.25.0
-      displayName: "Install CMake 3.25.0"
+        python -m pip install cmake
```

Comment:
is it ok to always get the latest? things could break unexpectedly. also, it might be hard to reproduce a build from an older commit if the cmake version used changes.

Reply:
Agree the points you mentioned here. But ort-extensions doesn't depend on any complex libraries anymore, so ideally latest cmake should be totally OK, unless there is very uncommon situation. 
Also, the fixed cmake version may requires the version update in the future as well. Furthermore, if some new cmake version causes some trouble, we can know it at the earliest time so that we can fix it quickly to help ort-extensions users which build it from source.

So, as a trade-off, keeping it be latest may be more suitable for ort-extensions project.